docs-gen/assets/scss/_styles_project.scss

```
@@ -0,0 +1 @@
+@import 'td/code-dark'
```

```suggestion
@import 'td/code-dark'

```
pro-end of file new line to prevent diff pollution